.github/workflows/plugin-cd.yml

```diff
@@ -29,6 +29,13 @@ on:
         required: false
         type: string
 
+      skip-aws-upload:
```

Comment:
For the CD part it's imperative to upload. This is how we download the pre-packaged plugins . 
Do we also need the flag there ?

Reply:
So we get the release on Github but not uploaded to S3, since the bundles are attached to releases here there's no need to have the S3 files around.

Reply:
When you bundle a plugin with mattermost platform currently it is being fetched from the S3 bucket through a cloudfront distribution . 
Since we have not yet changed this functionality we should continue uploading artifacts to S3 
https://github.com/mattermost/mattermost/blob/master/server/build/release.mk#L147

If you are completely sure that your plugin is not going to be bundled in the future with mattermost platform then you should use this but I suggest against that

Reply:
In this case this plugin won't be bundled since it will become EoL the moment the full feature arrives in Mattermost, and I see this being like that for a number of plugins: cc: @wiggin77 

Are we putting plugins in the bucket "just in case", are we doing any kind of cleanup of deprecated/deleted/outdated plugins?

Reply:
@fmartingr Thanks for taking the time to look at this. I made a comment in our internal discussion to clarify that we're not saving "just in case" but as main data store - https://hub.mattermost.com/private-core/pl/epzwcuorgjgpzxgrnjamfqhrcc

> are we doing any kind of cleanup of deprecated/deleted/outdated plugins?
No, for it's protected (data integrity, compliance, audit, etc) but can do so if requested.

Reply:
Understood. Removed the option from CD.